compiler/crates/relay-lsp/tests/rename/fixtures/rename_fragment_argument_from_definition.expected

```diff
@@ -0,0 +1,40 @@
+==================================== INPUT ====================================
+query MeQuery($include_firstname: Boolean!) {
+  me {
+    ...UserFragment @arguments(return_firstname: $include_firstname)
+  }
+}
+
+fragment UserFragment on User @argumentDefinitions(retu|rn_firstname: { type: "Boolean!" }) {
+  name @skip(if: $return_firstname)
+  ... @defer(if: $return_firstname) {
+    og_name: name
+  }
+  firstName(if: $return_firstname)
+}
+
+fragment CommentFragment on Comment {
+  author {
+    ...UserFragment @arguments(return_firstname: true)
+  }
+}
+==================================== OUTPUT ===================================
+query MeQuery($include_firstname: Boolean!) {
+  me {
+    ...UserFragment @arguments(RENAMED: $include_firstname)
+  }
+}
+
+fragment UserFragment on User @argumentDefinitions(RENAMED: { type: "Boolean!" }) {
+  name @skip(if: $return_firstname)
+  ... @defer(if: $RENAMED) {
+    og_name: name
+  }
```

Comment:
Any idea why the rename works for the argument on the custom directive but not for `@skip`? Is it treated differently by the Visitor?

Reply:
Nvm, I can get a handle on ot through visit_variable, but it feels not intuitive.